### PR TITLE
Fix manual testing script, allow access to /user/me by default

### PR DESCRIPTION
**Pull request by dscottboggs**
These changes are necessary to get the web interface working.

### CODE REVIEW

File: scripts/setup-for-manual-testing.sh [hunk 1]
```diff
@@ -32,12 +32,12 @@ echo '{"groups":[], "users": []}' > $DATA_DIR/permissions.json
...
 bin/dppm server group add "name=${DPPM_USER}s group" id=$GROUP_ID
-bin/dppm server add_user "name=$DPPM_USER" groups=$GROUP_ID
+bin/dppm server user add "name=$DPPM_USER" groups=$GROUP_ID
 
 if [ $ADMIN = yes ]; then
   make_admin
 fi
 
-bin/dppm server run
+bin/dppm server run host="127.0.0.1"
```
Comment (j8r) [inline scripts/setup-for-manual-testing.sh:41]: The default host is already localhost, `[::1]`

Comment (dscottboggs) [reply scripts/setup-for-manual-testing.sh:41]: I realize this, but it still did not work.

Edit: I mean to say, the webpack dev server proxy couldn't connect to the running dppm server

Comment (j8r) [reply scripts/setup-for-manual-testing.sh:41]: That's a webpack configuration to do, I suppose an option like `--host ::1`.

**Comment by j8r** [comment]
Script moved to https://github.com/DFabric/dppm-integration

**Pull request closed without merging**